src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs

```diff
@@ -4822,6 +4856,12 @@ protected virtual void OnSearchForVirtualItem(SearchForVirtualItemEventArgs e)
         protected virtual void OnSelectedIndexChanged(EventArgs e)
         {
             ((EventHandler)Events[EVENT_SELECTEDINDEXCHANGED])?.Invoke(this, e);
+
+            if (this.SelectedItems.Count > 0 && this.SelectedItems[0].Focused)
```

Comment:
We don't use `this.` unless absolutely necessary.
```suggestion
            if (SelectedItems.Count > 0 && SelectedItems[0].Focused)
```

The same below